docs/translations/de/client/ClientCommands.md

```diff
@@ -0,0 +1,44 @@
+---
+Titel: "Client Befehle"
```

Comment:
please don't convert the metadata tags, the system looks for `title` and `description` it doesn't translate struct field names as these are static and defined in code.

Reply:
Ah i see its changed already.
From now on, how do i go on about adding to the translations ? 
Do i simply add the files to this PR or how does it work?